android-sdk/android/src/test/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandlerTest.kt

```diff
@@ -59,7 +59,10 @@ class LogtoWebViewSocialHandlerTest {
                 window.logtoNativeSdk = {
                     platform: 'android',
                     getPostMessage: () => (data) => window.SocialHandler.postMessage(JSON.stringify(data)),
-                    supportedSocialConnectorTargets: ['wechat-native', 'alipay-native'],
+                    supportedConnector: {
+                        universal: true,
```

Comment:
Per offline discussion, not sure whether we should add a comment here for further explanation on setting universal to be true.